src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts

```diff
@@ -3,132 +3,548 @@ import { AfterViewInit, Component } from '@angular/core';
+            },
+          ],
+        },
+        {
+          label: 'No',
+          clickedContent: [],
+        },
+        {
+          label: 'I don’t know',
+          clickedContent: [],
+        },
+      ],
+      forwardButtons: [
+        {
+          label: 'Back',
+          name: 'back',
+        },
+        {
+          label: 'Next',
+          name: 'next',
+        },
+      ],
+    },
+    {
+      label:
+        'Are there any court applications underway between you and the other ...',
+      content: [
+        {
+          label:
+            ' Are there any court applications underway between you and the other parent that could have an impact on the amount of child support in your order or written agreement?',
+        },
+        {
+          label:
+            'For example, a party is applying to reduce or cancel child support, or someone is applying to change the parenting arrangements where a child will spend a different amount of time with each parent. ',
+        },
+      ],
+      buttons: [
+        {
+          label: 'Yes',
+          clickedContent: [
+            {
+              data: 'If you or the other parent have legal proceedings underway that may impact child support payments, the recalculation service <strong>cannot</strong> recalculate your order. Once you obtain a new order, you can apply to the Child Support Recalculation Service.',
+            },
+            {
+              data: 'The Child Support Recalculation Service recalculates child support in a less informal way than court. Parents enrolled in the service won\'t have to go to court and ask a judge to decide on child support when their income changes.',
+            },
+            {
+              data: '<strong>Important note:</strong> The Child Support Recalculation Service cannot backdate the recalculation of your child support. It can only recalculate child support amounts going forward based on income tax information for the most recent tax year.',
+            },
+            {
+              data: 'The recalculation service provides a copy of the recalculation statement to both parents and files a copy in court. Once it\'s in effect, the recalculated amount is the child support to be paid.',
+            },
+          ],
+        },
+        {
+          label: 'No',
+          clickedContent: [],
+        },
+      ],
+      forwardButtons: [
+        {
+          label: 'Back',
+          name: 'back',
+        },
+        {
+          label: 'Next',
+          name: 'next',
+        },
+      ],
+    },
+  ];
 
   constructor() {
-    this.bceIdLink = "https://www.bceid.ca/";
-   }
-
-  public ngAfterViewInit(): void {
-  }
-
-  // Question 1
-  q1Yes()
-  {
-    this.a1Yes = true; this.a1No = !this.a1Yes;
-  }
-  q1No()
-  {
-    this.a1No = true; this.a1Yes = !this.a1No;
+    this.bceIdLink = 'https://www.bceid.ca/';
```

Comment:
FYI:
We may have to pass/load environment specific values here.
The link is not same for dev, test and prod.